testing-utils/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
```

Comment:
The module `testing-utils` seems suspiciously similar to `data-report-test `. Merge these?

Reply:
My bad, I thought I had merged `data-report-test`, but I have not yet